blackbox/docs/cli-tools.rst

```diff
@@ -101,5 +108,111 @@ The CrateDB process can handle the following signals.
     the node is stopped, you can perform a `graceful stop`_ with the
     :ref:`DECOMMISSION <alter_cluster_decommission>` statement instead.
 
+Crate-node
+==========
+
+Managing clusters
+-----------------
+
+This section shows you how to manage CrateDB clusters from the command line.
+
+.. SEEALSO::
+
+   For step-by-step how-tos and examples on how to troubleshoot CrateDB clusters and
+   nodes from the command line, please refer to `Troubleshooting with crate-node CLI`_.
+
+Introduction
+------------
+
+CrateDB provides a ``crate-node`` command, in the ``bin`` directory, that lets
+you form new clusters, repair broken clusters, and repair broken nodes.
+
+With this command you can:
+
+ * Unsafely bootstrap a cluster
+ * Detach a node from its cluster
+ * Change the role of a node
+
+.. NOTE::
+
+  Cluster management operations on a node are only possible while the node is
+  shut down.
+
+A good way to start is to invoke ``crate-node`` with the ``-h`` option. This
+will give you an overview of the available options.
+
+.. code-block:: console
+
+    sh$ ./bin/crate-node --help
+
+Synopsis
+--------
+
+.. code-block:: console
+
+   bin/crate-node repurpose|unsafe-bootstrap|detach-cluster
```

Comment:
needs indenting by 4 chars

start command with `sh$`

if we're going to include this, we should standardize the syntax (preferably using some sort of commonly-used syntax) and also add a synopsis to the `crate` section. you will need an engineer for this. please create an impediment asking for help with this task